docs/agent-api.asciidoc

```diff
@@ -690,12 +690,12 @@ Optionally, a type may also be provided to group lambdas together. By default,
 Read more lambda support in the <<lambda,Lambda>> article.
 
 [[apm-add-patch]]
-==== `apm.addPatch(name, handler)`
+==== `apm.addPatch(modules, handler)`
 
 [small]#Added in: v2.7.0#
 
-* `name` +{type-string}+
```

Comment:
The type(s) within the curly braces needs to match a type defined in [`docs/index.asciidoc`](https://raw.githubusercontent.com/elastic/apm-agent-nodejs/master/docs/index.asciidoc) so that the link works.

We should probably do something along the lines of what we did with the `handler` argument below where there are two possible types listed, e.g:

```asciidoc
* `modules` +{type-string}+ | +{type-string-array}+
```

We don't yet have a `type-string-array` in `docs/index.asciidoc`, so we need to add that as well, e.g:

```asciidoc
:type-string-array: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type[<string[]>]
```

Reply:
Fixed